spacewalk/admin/spacewalk-startup-helper

```diff
@@ -113,6 +113,9 @@ check_database() {
     # Check, if the report DB was already setup
     if egrep -m1 "^report_db_host[[:space:]]*=[[:space:]]*[a-zA-Z0-9_-]+" /etc/rhn/rhn.conf; then
         REPORTDB_EXISTS='y'
+            if grep -E "^report_db_sslrootcert[[:space:]]*=[[:space:]]*/etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT" /etc/rhn/rhn.conf; then
+                sed -i "s|^report_db_sslrootcert[[:space:]]*=.*|report_db_sslrootcert = /etc/pki/trust/anchors/LOCAL-RHN-ORG-TRUSTED-SSL-CERT|" /etc/rhn/rhn.conf
+            fi
```

Comment:
As this is a onetime task, I would prefer to have it in the %post section of the spec file.
This one is executed at every restart of the spacewalk-servives, but %post section only on installation or update of the package.
And I think we should remove this in some month again as it should affect only uyuni users which hopefully update from time to time.